packages/browser/src/lib/backend.ts

```diff
@@ -112,15 +104,15 @@ export class BrowserBackend implements Backend {
     // both breadcrumbs created internally by Raven and pass them to the
     // Frontend first, before actually capturing them.
     Raven.setBreadcrumbCallback(breadcrumb => {
-      forget(this.frontend.addBreadcrumb(breadcrumb));
+      addBreadcrumb(breadcrumb);
```

Comment:
Implementing it like this would mean that you **could** be bleeding from one client into another. I think this should still go through this backend's own frontend to be safe. Also applies to the other functions below. Most importantly, this implementation means I can no longer just create an instance of the client without attaching it to the scope.

I understand the intention of doing this - to capture global exceptions and unhandled rejections in the currently active scope - but this is certainly the wrong place to do this.

Reply:
I do not think this applies here. Because there is only one raven and thus one callback so this should actually go into the currently active client.

Reply:
More importantly I think `install()` will go away entirely in the future when we kill `Raven`. The bridge stuff in react-native and co will not be in install anyways but a default instrumentation that cannot be disabled.

Reply:
Resolved offline, we make another PR which handles instrumentations separately.